db/migrate/20210918011908_add_gateway_session_id_to_order.rb

```diff
@@ -0,0 +1,5 @@
+class AddGatewaySessionIdToOrder < ActiveRecord::Migration[6.1]
+  def change
+    add_column :spree_orders, :gateway_checkout_session_id, :string
```

Comment:
If we're adding this, maybe `stripe_session_id` would be clearer. It wouldn't ever be used with other payment gateways.